File: Programmers/문자열 내 p와 y의 개수.cpp
#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{
    bool answer = true;
    int a = 0, b = 0;
    for (int i = 0; i < s.size(); i++)
    {
        if (s[i] == 'p' || s[i] == 'P') a += 1;
        if (s[i] == 'y' || s[i] == 'Y') b += 1;
    }
    if (a == b) answer = true;
    else answer = false;

    return answer;
}

int main() {
    string s;
    cin >> s;
    cout << solution(s);
    system("pause");
}

File: Programmers/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        int a = 0, b = 0;
        char[] tmp = s.toCharArray();
        for(int i : tmp)
            if (i=='p'||i=='P')
                a++;
            else if(i=='y' || i=='Y')
                b++;
        if(a==b)
            return true;
        else
            return false;
    }
}